tests/test_cli.py

```diff
@@ -196,21 +242,26 @@ def test_cli_output_modulecred_file(runner):
             os.remove(output_file_path)
 
 
-@pytest.mark.skipif(docker_client.get_os_type().lower() == 'windows', reason='It does not support windows container')
+@pytest.mark.skipif(get_docker_os_type() == 'windows', reason='It does not support windows container')
 def test_cli_create_options_for_custom_volume(runner):
     try:
         cli_setup(runner)
         remove_docker_volumes(['testVolume'])
-        cli_start_with_deployment(runner, 'deployment_with_custom_volume.json')
+        deployment_json_file_path = os.path.join(test_resources_dir, 'deployment_with_custom_volume.json')
+        cli_start_with_deployment(runner, deployment_json_file_path)
         wait_verify_docker_output(['docker', 'volume', 'ls'],
                                   ['testVolume', 'edgemoduledev', 'edgehubdev'])
         expected_volumes = (['"Source": "testVolume"',
                              '"Destination": "/mnt_test"',
                              '"Source": "edgemoduledev"',
                              '"Destination": "/mnt/edgemodule"'])
         wait_verify_docker_output(['docker', 'inspect', 'tempSensor'], expected_volumes)
-        wait_verify_docker_output(['docker', 'inspect', 'edgehubdev'],
-                                  '"Mountpoint": "/var/lib/docker/volumes/edgehubdev/_data"')
+        if platform.system().lower() == "linux":
+            wait_verify_docker_output(['docker', 'inspect', 'edgehubdev'],
+                                      ['"Mountpoint": "/var/lib/docker/volumes/edgehubdev/_data"'])
+        else:
+            wait_verify_docker_output(['docker', 'inspect', 'edgehubdev'],
```

Comment:
what system should be this path? better to specify the system, if more system are supported, the case will fail.

Reply:
I expect it failed when new system occurs. It's we want to know to update test case.

Reply:
Updated code